src/www/app/app.ts

```diff
@@ -296,6 +296,8 @@ export class App {
         return console.debug('Ignoring access key');
       } else if (fromClipboard && addServerView.isAddingServer()) {
         return console.debug('Already adding a server');
+      } else if (fromClipboard && accessKey.startsWith('https://')) {
```

Comment:
I think we do still want to auto-accept the Amazon s3 access keys that we have special case logic for.

Reply:
What's the format for that?

Reply:
FWIW the invite flow is about to be removed from production in 1.8.1

Reply:
That's here: https://github.com/Jigsaw-Code/outline-client/blob/c1facfc930353ab6ed26941a99eccf9714d729fd/config.xml#L68

But really the best thing is probably to call [`unwrapInvite()`](https://github.com/Jigsaw-Code/outline-client/blob/07b9a6694094000d2700fb899e31e617a00ae697/src/www/app/app.ts#L33) and ignore any non-`ss:` URL that doesn't unwrap successfully.

Reply:
Ben, it seems we are already calling unwrap above.

Reply:
True.  So those URLs should already have been converted to `ss://` by this point, so any `https://` URLs here should indicate failure to unwrap.

Reply:
So that means this code should be fine right? (in theory)